ports/mimxrt/boards/MIMXRT1010_EVK/pins.csv

```diff
@@ -0,0 +1,57 @@
+D0,GPIO_09
+D1,GPIO_10
+D2,GPIO_AD_05
+D3,GPIO_AD_06
+D4,GPIO_08
+D5,GPIO_01
+D6,GPIO_AD_01
+D7,GPIO_AD_02
+D8,GPIO_SD_02
+D9,GPIO_03
+D10,GPIO_AD_05
+D11,GPIO_AD_04
+D12,GPIO_AD_03
+D13,GPIO_AD_06
+D14,GPIO_01
+D15,GPIO_02
+CUR_A,GPIO_AD_01
+CUR_B,GPIO_AD_02
+CUR_C,GPIO_AD_07
+VOLT_DCB,GPIO_AD_09
+CUR_DCB,GPIO_AD_10
+A0,GPIO_AD_07
+A1,GPIO_AD_09
+A2,GPIO_AD_10
+A3,GPIO_AD_14
+A4,GPIO_AD_01
+A5,GPIO_AD_02
+PWM_AT,GPIO_02
+PWM_AB,GPIO_01
+PWM_BT,GPIO_04
+PWM_BB,GPIO_03
+PWM_CT,GPIO_06
+PWM_CB,GPIO_05
+ENC_A,GPIO_AD_05
+ENC_B,GPIO_AD_06
+LED_GREEN,GPIO_11
+GPIO_01,GPIO_01
```

Comment:
I think the `make-pins.py` script supports the notation that you can leave the first name out if it's the same as the second, eg `,GPIO_01`.

Reply:
Ok, I guess I have to modify my script for that. I have basically written it from scratch to get a better grasp.

Reply:
Ah, I see.  Well, it should be easy to add this feature.  And I think it's quite useful to indicate which pins don't have a special name (just the datasheet name) but are still exposed to the user.

Reply:
Yes. I will fix that. 👍